app/models/server/models/Users.js

```diff
@@ -172,6 +172,57 @@ export class Users extends Base {
 		return null;
 	}
 
+	async getNextLeastBusyAgent(department) {
```

Comment:
please use the specific raw model for raw methods: https://github.com/RocketChat/Rocket.Chat/blob/develop/app/models/server/raw/Users.js